.github/workflows/create-azure-machine.ps1

```diff
@@ -48,6 +48,8 @@ if ($Os -eq "windows-2025") {
     $image = "MicrosoftWindowsServer:WindowsServer:2022-datacenter-g2:latest"
 } elseif ($Os -eq "windows-2019") {
     $image = "MicrosoftWindowsServer:WindowsServer:2019-datacenter-gensecond:17763.5576.240304"
+} elseif ($Os -eq "ubuntu-24.04") {
```

Comment:
FYI this script has been re-purposed to be used as a manual utility tool for quickly spinning  up a test VM using Azure CLI. We are rotating this out of the pipeline due to security reasons. We don't need to worry about updating this.